Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.6-slim
+
+VOLUME /roboflow
+WORKDIR /roboflow
+COPY README.md setup.py /roboflow/
+
+# Add GCC + CV2 dependencies
+RUN apt-get update \
+    && apt-get -y install python3-dev build-essential ffmpeg libsm6 libxext6
+
+RUN pip3 install -U pip \
```

Comment:
If you want to dev locally, you don't have to copy the files over, you will just install it once after you run the container

Reply:
this is true, but then you have to install dependencies every time you start the container. i personally like to automate this upon build to save time while developing later.

Reply:
Replied on slack :)